homeassistant/components/playstation_network/strings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "npsso": "NPSSO token"
+        },
+        "data_description": {
+          "npsso": "Please supply the NPSSO token generated during successful login of your PlayStation Network account"
```

Comment:
The data description isn't a question, it's rather an explanation of the data you need